examples/topic/cruise-control-topics.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: kafka.strimzi.io/v1beta1
+kind: KafkaTopic
+metadata:
+  name: strimzi.cruisecontrol.partitionmetricsamples
+  labels:
+    strimzi.io/cluster: my-cluster
+spec:
+  partitions: 32
+  replicas: 2
+  config:
+    cleanup.policy: delete
+    retention.ms: "3600000"
+---
+apiVersion: kafka.strimzi.io/v1beta1
+kind: KafkaTopic
+metadata:
+  name: strimzi.cruisecontrol.modeltrainingsamples
+  labels:
+    strimzi.io/cluster: my-cluster
+spec:
+  partitions: 32
+  replicas: 2
+  config:
+    cleanup.policy: delete
+    retention.ms: "12000000"
+---
+apiVersion: kafka.strimzi.io/v1beta1
+kind: KafkaTopic
+metadata:
+  name: strimzi.cruisecontrol.metrics
+  labels:
+    strimzi.io/cluster: my-cluster
+spec:
+  partitions: 1
+  replicas: 1
+  config:
```

Comment:
If providing the KafkaTopic resource is a workaround for the bug in the CC metrics reporter I think that only this strimzi.cruisecontrol.metrics is needed here to be created. The other ones should be created by the CC not the metrics reporter. Can you double check @kyguy?

Reply:
You are correct; I have just updated the template file accordingly